embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/config/OkHttpRemoteConfigSource.kt

```diff
@@ -41,8 +43,11 @@ internal class OkHttpRemoteConfigSource(
         if (!response.isSuccessful) {
             return null
         }
-        val config = response.body?.source()?.inputStream()?.buffered()?.use {
-            serializer.fromJson(it, RemoteConfig::class.java)
+        val config = response.body?.source()?.use { src ->
```

Comment:
Is the tiny payload we get back worth gzipping? Or would it be annoying and not worthwhile do do something different here?

Reply:
I'm easy either way - let's discuss internally & update in a separate PR if necessary